api/api-doc/storage_service.json

```diff
@@ -701,6 +701,30 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/compact",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Forces major compaction in all keyspaces",
```

Comment:
To be honest, I don't understand when this API is ever useful. Maybe just when you have exactly one big table?
I remember in the past we tried to claim that users should never explicitly call major compaction at all - that it just ruins performance. So now we're giving them a way to do a major compaction of all tables at once?

Reply:
In dtest (that represents users mindset) bare `nodetool compact`, that causes major compaction of all keyspaces, is used elaborately, and naive users may also invoke it sparringly.
Unfortunately in many use cases major compaction is still required once in a while to prevent accumulation of deleted data and tombstones, though in this case I admit the usage may typically be needed for a specific table or keyspace.
In any case, the API is there also to help field-engineering in extreme corner cases.  Having an API for something doesn't mean we're encouraging anyone to use it on a regular basis.  It's a power-tool that should be used with care, understanding what one is doing.